Dockerfile

```diff
@@ -22,6 +22,6 @@ RUN pip install --user -r requirements.txt
 
 COPY setup.py ./
 COPY phabricatoremails/ phabricatoremails/
-RUN pip install --user .
+RUN pip install --user -e .
```

Comment:
A better way of fixing the problem of `settings.ini` not being copied and accessible when installing this package would be to include it in `package_data` in your `setup.py`. See https://docs.python.org/3/distutils/setupscript.html#distutils-installing-package-data. `pip -e` is intended for development use only.

Reply:
> pip -e is intended for development use only.

Hmm, I was worried about that, good point.

> A better way of fixing the problem of settings.ini not being copied and accessible when installing this package would be to include it in package_data in your setup.py

Unfortunately, `settings.ini` isn't available at docker-build time, but is rather injected into the container at runtime using a docker volume.

One possible alternative is that I can remove the `setuptools` information for `phabricator-emails`, and just have a python file that is invoked directly with:
```
if __name__ == __main__:
    cli()
```
Although, I like the convenience of having setuptools create the entrypoints for us. Do you know if we have a better alternative, or should I implement :point_up: ?

Reply:
I see -- can you post the full stack trace from that original error?

```Exception: No config file found at "/app/.local/lib/python3.8/site-packages/settings.ini"```

Reply:
Sure thing:
```
Jun 26 18:32:24 ip-172-31-42-230.us-west-2.compute.internal docker-phabricatoremails[4590]: Traceback (most recent call last):
Jun 26 18:32:24 ip-172-31-42-230.us-west-2.compute.internal docker-phabricatoremails[4590]: File "/app/.local/bin/phabricator-emails", line 8, in <module>
Jun 26 18:32:24 ip-172-31-42-230.us-west-2.compute.internal docker-phabricatoremails[4590]: sys.exit(cli())
Jun 26 18:32:24 ip-172-31-42-230.us-west-2.compute.internal docker-phabricatoremails[4590]: File "/app/.local/lib/python3.8/site-packages/phabricatoremails/cli.py", line 35, in cli
Jun 26 18:32:24 ip-172-31-42-230.us-west-2.compute.internal docker-phabricatoremails[4590]: settings = Settings.load()
Jun 26 18:32:24 ip-172-31-42-230.us-west-2.compute.internal docker-phabricatoremails[4590]: File "/app/.local/lib/python3.8/site-packages/phabricatoremails/settings.py", line 133, in load
Jun 26 18:32:24 ip-172-31-42-230.us-west-2.compute.internal docker-phabricatoremails[4590]: raise Exception(f'No config file found at "{path}"')
Jun 26 18:32:24 ip-172-31-42-230.us-west-2.compute.internal docker-phabricatoremails[4590]: Exception: No config file found at "/app/.local/lib/python3.8/site-packages/settings.ini"
```

Reply:
Was spitballing with Aki today and he mentioned that we could also solve this by having an optional environment variable that configured where the `settings.ini` would go. Worth :thinking: about, but let me know what you'd prefer :)

Reply:
If they are injected at runtime, I think the approach of setting an environment variable to specify the path is fine.